File: dongyimai_manager_web/src/main/webapp/js/service/itemService.js
//商品表服务层
app.service('itemService',function($http){
	    	
	//findAll
	this.findAll=function(){
		return $http.get('../item/findAll.do');		
	}

	//findPage
	this.findPage=function(page,rows){
		return $http.get('../item/findPage.do?page='+page+'&rows='+rows);
	}

	//findOne
	this.findOne=function(id){
		return $http.get('../item/findOne.do?id='+id);
	}

	//add
	this.add=function(entity){
		return  $http.post('../item/add.do',entity );
	}

	//update
	this.update=function(entity){
		return  $http.post('../item/update.do',entity );
	}

	//dele
	this.dele=function(ids){
		return $http.get('../item/delete.do?ids='+ids);
	}

	//条件、模糊查询+分页
	this.search=function(page,rows,searchEntity){
		return $http.post('../item/search.do?page='+page+"&rows="+rows, searchEntity);
	}

});

File: dongyimai_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/ItemService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbItem;

import java.util.List;

public interface ItemService {

	//findAll
	public List<TbItem> findAll();
	
	//findPage
	public PageResult findPage(int pageNum, int pageSize);
	
	//add
	public void add(TbItem item);
	
	//update
	public void update(TbItem item);
	
	//findOne
	public TbItem findOne(Long id);
	
	//delete
	public void delete(Long[] ids);

	//条件、模糊查询+分页
	public PageResult findPage(TbItem item, int pageNum, int pageSize);
	
}
